whitenoise/base.py

```diff
@@ -11,6 +12,8 @@
 from .string_utils import (decode_if_byte_string, decode_path_info,
                            ensure_leading_trailing_slash)
 
+TEXT_TYPE = str if sys.version_info[0] >= 3 else unicode
+
```

Comment:
Maybe you want move them to `string_utils.py` as `BINARY_TYPE` ?

https://github.com/evansd/whitenoise/blob/a51cc1a70e02f45cf0a02d1f9ed4d5d42d946986/whitenoise/string_utils.py#L4

Reply:
Good suggestion!